language/golang/run-containers.md

```diff
@@ -88,7 +88,7 @@ Hello, Docker! <3
 
 ## List containers
 
-Since we ran our container in the background, how do we know if our container is running or what other containers are running on our machine? Well, we can run the `docker ps` command. Just like on Linux, to see a list of processes on your machine we would run the `ps` command. In the same spirit, we can run the `docker ps` command which will show us a list of containers running on our machine.
+Since we ran our container in the background, how do we know if our container is running or what other containers are running on our machine? Well, to see a list of containers running on our machine, run `docker ps`. This is similar to how ps command is used to see a list of processes on a Linux machine.
```

Comment:
Sorry, I didn't catch this in the previous PR for node.js.  If possible, can you update the node.js topic in this PR too.
I think it should have `the` or remove `command`.
For example 
 * `...how the ps command is used...` 
 * or `...how ps is used...`

```suggestion
Since we ran our container in the background, how do we know if our container is running or what other containers are running on our machine? Well, to see a list of containers running on our machine, run `docker ps`. This is similar to how the ps command is used to see a list of processes on a Linux machine.
```